lib/config.js

```diff
@@ -82,6 +84,7 @@ Config.prototype.buildArgs = function () {
     brave_version_minor: version_parts[1],
     brave_version_build: version_parts[2],
     chrome_version_string: this.chromeVersion,
+    chrome_version_major: chrome_version_parts[0],
```

Comment:
During the installer build, different style of brave full version is used like `70.0.58.1` or `70_0_58_1`. 
To make these strings easily in gn, pass chromium's major version number.